### PR TITLE
[docs] Update FAQ about reducing master nodes

### PR DESCRIPTION
**Pull request by Zhbert**
## Description

Improve documentation on reducing the number of master nodes in the cluster.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: docs
type: fix
summary: Improved documentation on reducing the number of master nodes in the cluster.
impact_level: low
```

### CODE REVIEW

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 1]
```diff
@@ -91,6 +95,15 @@ title: "Управление control plane: FAQ"
      --ssh-user=<USERNAME> --ssh-host <MASTER-NODE-0-HOST>
    ```
 
+   > Для **Yandex Cloud** при использовании внешних адресов на master-узлах количество элементов массива в параметре [masterNodeGroup.instanceClass.externalIPAddresses](../030-cloud-provider-yandex/cluster_configuration.html#yandexclusterconfiguration-masternodegroup-instanceclass-externalipaddresses) должно равняться количеству master-узлов. При использовании значения `Auto` (автоматический заказ публичных IP-адресов) количество элементов в массиве все равно должно соответствовать количеству master-узлов.
+   >
+   > Например, при одном master-узле (`masterNodeGroup.replicas: 1`) и автоматическом заказе адресов параметр `masterNodeGroup.instanceClass.externalIPAddresses` будет выглядеть следующим образом:
+   >
+   > ```bash
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:102]:
```suggestion
   > ```yaml
```

File: modules/040-control-plane-manager/docs/FAQ_RU.md [hunk 2]
```diff
@@ -71,6 +71,10 @@ title: "Управление control plane: FAQ"
 
 ## Как уменьшить число master-узлов в облачном кластере (multi-master в single-master)?
 
+{% alert level="warning" %}
+Описанные ниже шаги необходимо выполнять с первым master-узлом кластера (master-0). Это связано с тем, что кластер всегда масштабируется по порядку: например, невозможно удалить узлы master-0 и master-1, оставив master-2.
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ_RU.md:75]:
```suggestion
Описанные ниже шаги необходимо выполнять с нулевым master-узлом кластера (master-0). Это связано с тем, что кластер всегда масштабируется по порядку: например, невозможно удалить узлы master-0 и master-1, оставив master-2.
```

Comment (z9r5) [reply modules/040-control-plane-manager/docs/FAQ_RU.md:75]: Или может  - с первого по порядку?

File: modules/040-control-plane-manager/docs/FAQ.md [hunk 3]
```diff
@@ -99,6 +103,15 @@ Adding a master node to a static or hybrid cluster has no difference from adding
      --ssh-user=<USERNAME> --ssh-host <MASTER-NODE-0-HOST>
    ```
 
+   > For **Yandex Cloud**, when using external addresses on master nodes, the number of array elements in the [masterNodeGroup.instanceClass.externalIPAddresses](../030-cloud-provider-yandex/cluster_configuration.html#yandexclusterconfiguration-masternodegroup-instanceclass-externalipaddresses) parameter must equal the number of master nodes. If `Auto` is used (public IP addresses are provisioned automatically), the number of array elements must still equal the number of master nodes.
+   >
+   > To illustrate, with three master nodes (`masterNodeGroup.replicas: 1`) and automatic address reservation, the `masterNodeGroup.instanceClass.externalIPAddresses` parameter would look as follows:
+   >
+   > ```bash
```
Comment (z9r5) [inline modules/040-control-plane-manager/docs/FAQ.md:110]:
```suggestion
   > ```yaml
```

**Review by z9r5 — COMMENTED** [review]
comments

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10272](https://github.com/deckhouse/deckhouse/pull/10272) to the branch [release-1.65](https://github.com/deckhouse/deckhouse/tree/release-1.65) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [10273](https://github.com/deckhouse/deckhouse/pull/10273) to the branch [release-1.64](https://github.com/deckhouse/deckhouse/tree/release-1.64) successful!

**Pull request merged**